Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -211,7 +211,84 @@ public function isArrayMultiLine($index)
     }
 
     /**
-     * Check if there is a lambda function under given index.
+     * Returns the attributes of the method under the given index.
+     *
+     * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array {
+     *     @var int|null $visibility T_PRIVATE, T_PROTECTED or T_PUBLIC
```

Comment:
@GrahamCampbell does this `@var` should be considered as annotation?
`PhpdocVarWithoutNameFixer` fixer removes names here, but it should not because they are valid PHPDoc